drivers/include/mma8652.h

```diff
@@ -104,7 +116,7 @@ int mma8652_test(mma8652_t *dev);
  * @return                  -4 if setting to STANDBY mode failed
  * @return                  -5 if accelerometer configuration failed
  */
-int mma8652_init(mma8652_t *dev, i2c_t i2c, uint8_t address, uint8_t dr, uint8_t range);
+int mma8652_init(mma8652_t *dev, i2c_t i2c, uint8_t address, uint8_t dr, uint8_t range, uint8_t type);
```

Comment:
Don't you think it's a bit confusing to stay with the `mma8`**6**`5`**2**` while changing the driver to `mma8x5x`? Was it because of the API change?